MyJobApplication/src/com/amaderorg/myjobapp/Presenter/Contacts/IContactInformationPresenter.java

```diff
@@ -0,0 +1,22 @@
+package com.amaderorg.myjobapp.Presenter.Contacts;
+
+/**
+ * Created by souvi_000 on 3/28/2016.
+ */
+public interface IContactInformationPresenter {
+
+    /**
+     * Adding sender information to the database
+     *
+     * @param firstName : first name
+     * @param lastName  : last name
+     * @param company   : company name
+     * @param location  : location
+     * @param email     : email id
+     */
+    void addSenderInformation(String firstName,
```

Comment:
Since the 'Contact' class is public, it can be used directly at this level. Whoever is using this presenter can create a Contact object and pass it in this method.